dev/TeachingTip/TeachingTip.cpp

```diff
@@ -1552,7 +1552,8 @@ void TeachingTip::CreateContractAnimation()
     m_contractElevationAnimation.set([this, compositor, contractEasingFunction]()
     {
         auto const contractElevationAnimation = compositor.CreateVector3KeyFrameAnimation();
-        contractElevationAnimation.InsertExpressionKeyFrame(1.0f, L"Vector3(this.Target.Translation.X, this.Target.Translation.Y, 0.0f)", contractEasingFunction);
+        // animating to 0.01f instead of 0.0f as work around to internal issue 26001712 which was causing text clipping.
+        contractElevationAnimation.InsertExpressionKeyFrame(1.0f, L"Vector3(this.Target.Translation.X, this.Target.Translation.Y, 0.01f)", contractEasingFunction);
```

Comment:
Out of curiosity, do we know why this "fixes" the bug? 😅

Reply:
It has to do with an over optimization that's happening in comp. The team is still looking into how we are getting into the bad state but its something around the transition from a 2d object to a 3d object.

Reply:
Oh okay, thanks for the info!